src/event_loop.rs

```diff
@@ -301,6 +301,19 @@ impl<T> EventLoopWindowTarget<T> {
     self.p.set_progress_bar(_progress)
   }
 
+  /// Sets the badge count
+  ///
+  /// ## Platform-specific
+  ///
+  /// - **Windows:** Unsupported. Use the Progress Bar Function Available in Window (Windows can have different taskbar badges for different Window)
+  /// - **Linux:** Only supported desktop environments with `libunity` (e.g. GNOME).
```

Comment:
I think we can support it on iOS, see https://github.com/tauri-apps/tauri/issues/4489#issuecomment-2276804628

Reply:
Implemented, please review